packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -1,15 +1,15 @@
-// @ts-nocheck
-/*
- * This program and the accompanying materials are made available under the terms of the *
- * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
- * https://www.eclipse.org/legal/epl-v20.html                                      *
- *                                                                                 *
- * SPDX-License-Identifier: EPL-2.0                                                *
- *                                                                                 *
- * Copyright Contributors to the Zowe Project.                                     *
- *                                                                                 *
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
  */
 
+// @ts-nocheck
```

Comment:
Curious why we use `@ts-nocheck` - feels like it could lead to undetected code smells?

Reply:
thanks @t1m0thyj these are things we can take care of either on main or as we refactor and add new to v3's next branch.

Reply:
addressed in 18f1972ed92a4328ea933a6579ba15efe2665da6
Thanks @traeok